README.md

```diff
@@ -16,10 +16,31 @@ To build, install Stata locally, apply your licence, and then copy the
 following into the `bin/` subdirectory:
 
 * `/usr/local/stata/bin/stata*`
+* `/usr/local/stata/bin/stinit`
 * `/usr/local/stata/bin/ado/`
 * `/usr/local/stata/utilities/`
 
+
 ## Relicensing Stata
 
-To relicense, apply the licence locally, and then copy
-`/usr/local/stata/stata.lic` into `bin/`.
+The docker image contains the machinery needed to update the license. To
+generate a new license file, first obtain the new serial number, code, and
+authorisation. Then run:
```

Comment:
Worth including some examples of what each one should look like? (For example, I'm thinking the serial number probably has to include spaces?)

Reply:
Good, point, and excellent example, as it's `code` that has spaces!

Have added explicit examples